presto-hive-hadoop2/pom.xml

```diff
@@ -157,4 +163,30 @@
             </build>
         </profile>
     </profiles>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-core</artifactId>
+                <version>1.11.333</version>
```

Comment:
Please use ${dep.aws-sdk.version} instead of hard coding the version in each place. You are giving value for dep.aws-sdk.version on line 50.